src/index.ts

```diff
@@ -127,6 +127,7 @@ module.exports = class ServerlessPlugin {
           aws,
           datadogForwarderArn,
           config.integrationTesting,
+          config.subscribeToApiGatewayLogs,
```

Comment:
Should you grab the other new  parameters also?
  `subscribeToHttpApiLogs`
  `subscribeToWebsocketLogs`
I think you implemented logic that defaults these parameters to true however does the logic support if the customer wants to opt out. Ie. if they explicitly say this in there serverless.yml:
```
subscribeToApiGatewayLogs: false
subscribeToHttpApiLogs: false
subscribeToWebsocketLogs: false
```